martian-pipestance/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+use anyhow::{Context, Result};
```

Comment:
I think this crate should expose an explicit API by `pub use`ing the relevant types/functions.